### DISCUSSION TITLE
Saw this today on Visual Studio Magazine website

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1079] [General]
thought I would share.
[https://visualstudiomagazine.com/articles/2021/01/21/oqtane.aspx](https://visualstudiomagazine.com/articles/2021/01/21/oqtane.aspx)

**Comment by hishamco** [comment]
The `href` of the above url was broken it should be the same ;)

Thanks for sharing, but I think @sbwalker already descibed this in one of the Oqtane blog posts

**Reply by leigh-pointer** [reply]
I have fixed the URL.
The news post was By David Ramel on 01/21/2021

**Reply by sbwalker** [reply]
Thanks for posting Leigh. I was surprised to see the article as I have never met David Ramel. I think he wrote the article because Oqtane was the .NET Foundation featured project in December. It was interesting to read his observations.
